scripts/rebuildstaging

```diff
@@ -52,7 +52,7 @@ done
 
 function rebuildstaging() {
     echo "rebuilding staging branch, this might take a while..."
-    python scripts/rebuildstaging.py < scripts/staging.yaml "$@"
+    python scripts/rebuildstaging.py scripts/staging.yaml "$@"
```

Comment:
Curious, what does this change do?

Reply:
looks like its now passing the file path instead of file content.. some shell magic..

Reply:
👍  thanks

Reply:
Previously it was streaming the file content via stdin, now it's passing the file path. It's hard to use pdb when the process reads stdin.